hnn_core/cell.py

```diff
@@ -443,6 +529,9 @@ def build(self, sec_name_apical=None):
             with this section. The section should belong to the apical dendrite
             of a pyramidal neuron.
         """
+        # Confirm correctness here
+        from .network_builder import load_custom_mechanisms
+        load_custom_mechanisms()
```

Comment:
does it need to be in the other place now that you have the function call here?

Reply:
I'll try removing it from the other place